cdc/api/v2/changefeed.go

```diff
@@ -176,7 +170,7 @@ func (h *OpenAPIV2) createChangefeed(c *gin.Context) {
 
 	log.Info("Create changefeed successfully!",
 		zap.String("id", info.ID),
-		zap.String("changefeed", infoStr))
+		zap.String("changefeed", info.String()))
```

Comment:
`changefeedInfo.String()` should be modify to mask all possible credentials information.

Reply:
fixed